rotkehlchen/tests/unit/globaldb/test_upgrades.py

```diff
@@ -645,6 +672,32 @@ def test_upgrade_v7_v8(globaldb: GlobalDBHandler):
             ('SPAM_ASSET_FALSE_POSITIVE', 'eip155:1/erc20:0xA0b73E1Ff0B80914AB6fe0444E65848C4C34450b'),  # noqa: E501
         ).fetchone()[0] == 1
 
+        # check that asset_collections have unique entries now with correctly mapped ids
+        assert table_exists(
+            cursor=cursor,
+            name='asset_collections',
+            schema=DB_CREATE_ASSET_COLLECTIONS,
+        ) is True
+
+        for collection_id, name, symbol in (
+            (260, 'Moon App', 'APP'),
+            (261, 'Starknet', 'STRK'),
+            (262, 'Wormhole Token', 'W'),
+            (263, 'TokenFi', 'TOKEN'),
+            (264, 'HTX', 'HTX'),
+            (300, 'Kyber Network Crystal v2', 'KNC'),
+            (301, 'PancakeSwap Token', 'CAKE'),
+        ):
+            assert cursor.execute(
+                'SELECT COUNT(*) FROM asset_collections WHERE id=? AND name=? AND symbol=?',
```

Comment:
this checks also for the combination of id. Can you check that the name/symbol aren't duplicated?